lib/src/ext/KHR_materials_volume/khr_materials_volume.dart

```diff
@@ -71,6 +71,21 @@ class KhrMaterialsVolume extends GltfProperty {
       thicknessTexture.link(gltf, context);
       context.path.removeLast();
     }
+
+    if (!context.validate) {
+      return;
+    }
+
+    Object o = this;
+    while (o != null) {
+      o = context.owners[o];
+      if (o is Material) {
+        if (!o.extensions.containsKey(KHR_MATERIALS_TRANSMISSION)) {
```

Comment:
Can we allow unknown material extensions to suppress this warning?  There's a draft KHR_materials_translucency that may get renamed to KHR_materials_diffuse_transmission, either of which would dismiss the need for KHR_materials_transmission.  It's also possible (though unlikely) that a vendor-specific extension could allow light through the surface of a volume.

Reply:
The plan is to add transmission-related KHR and EXT extensions here once they become stable. Suppressing the message when a vendor-specific extension is present sounds fine.

Reply:
Looking into this again, we should suppress the warning regardless of the extension prefix when an unknown material extension is present.